sdk/examples/how_tos/native_tokens/burn.rs

```diff
@@ -32,15 +32,14 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
+    let alias = "Alice";
```

Comment:
remove?

Reply:
either I missed it, or it was reintroduced accidentally during a merge :shrug:

Reply:
Done